astropy/table/column.py

```diff
@@ -111,8 +88,8 @@ def __setslice__(self, start, stop, val):
                              .format(self.__class__.__name__))
 
 
-class ColumnInfo(DataInfo):
-    attrs_from_parent = DataInfo.attr_names
+class ColumnInfo(BaseColumnInfo):
```

Comment:
Not a big deal, but is there still a need to define `ColumnInfo` at all? Since you are defining `BaseColumn.info` below, I think it would be fine for it just to be `info = InfoDescriptor(BaseColumnInfo)`.